Makefile

```diff
@@ -30,6 +30,9 @@ docs: .paasta/bin/activate
 test: .paasta/bin/activate
 	.paasta/bin/tox -i $(PIP_INDEX_URL)
 
+api: .paasta/bin/activate
+	python3.6 ./example_cluster/setup_fake_config.py
```

Comment:
This assumes the virtualenv is active.
Instead call `.paasta/bin/python ./example...`